### PR TITLE
Add default metadata help for collection fixes #1057

### LINKED ISSUE #1057: Sufia should default metadata help for collection
In ScholarSphere we were getting empty help on collection editing form.  Sufia should default it like it defaults GenericFile's help.

We added help in SS here:
https://github.com/psu-stewardship/scholarsphere/commit/bccb0fb86596fe1af5fefed40293b799479b7b08

### PR DESCRIPTION
**Pull request by yinlinchen**

### CODE REVIEW

File: config/locales/sufia.en.yml [hunk 1]
```diff
@@ -204,6 +204,8 @@ en:
         language: "The language of the file content."
         publisher: "The person or group making the file available. Generally this is the institution."
         rights: "Licensing and distribution information governing access to the file. Select from the provided drop-down list. &lt;em&gt;This is a required field&lt;/em&gt;."
+      collection:
+        creator_html: "The person or group responsible for the collection. Usually this is the author of the collected content. Personal names should be entered with the last name first, e.g. &quot;Smith, John.&quot; &lt;em&gt;This is a required field&lt;/em&gt;."
```
Comment (awead) [inline config/locales/sufia.en.yml]: If you're using `_html` in the yaml key, you should be able to write html directly.

Comment (cam156) [inline config/locales/sufia.en.yml]: @yinlinchen what @awead  is saying is you can put in "Smith, John" instead of &quot;Smith, John.&quot;  ect.

Comment (awead) [inline config/locales/sufia.en.yml]: @yinlinchen yes, here's an example https://github.com/projecthydra/sufia/blob/master/config/locales/sufia.en.yml#L69

**Comment by cam156** [comment]
:+1:  Waiting on Travis

**Pull request merged**